### ISSUE TITLE
not working for me

### ISSUE THREAD

**Issue by megusd** [Whizkidzz/Solaredge#2]
Hi there
Thank you for inverter configuration v2 (btw I can't find any other version now, seems wiped out of the net)
I have tried to generate pass codes for my serial number and put them in to configuration tool connected to  y inverter but configuration tool wont accept it. Says "If you are a technician and having troubles to log in, please ask for help by email support@solaredge.com"

**Comment by Whizkidzz** [comment]
For your inverter id only use capitals
Then it should work

**Comment by megusd** [comment]
I have tried all possible combinations of my serial: capitals/numbers, with/without "-"
Nothing helps

**Comment by Whizkidzz** [comment]
Then i An out of options
Normally i copy and paste it

**Comment by megusd** [comment]
Hi Anton
Just to confirm. I type "Tech" in the name field and copy/past generated password, right?

Do you have by a chance configuration software v3.0? I suspect if my inverter too new it may not work with v2.0 correctly..

Dmitry

**Comment by Whizkidzz** [comment]
In the name/id field comes the device id in capitals
And paste the tech or tech-adv password

**Comment by megusd** [comment]
Yes, I understand to put inverter ID in password generator then copy/past one of the passwords in to configuration tool . But what to use in "User" field in access control dialog of configuration tool??

Also do you keep zero at the beginning of the device ID to generate passwords?

Dmitry

**Comment by Whizkidzz** [comment]
The device id is XXXXXXXX-XX 
So always a serial Number of 8 hex digits and a '-' and a checksum of 2 hex digits
So if your serial begins with a 0 then it must be included

And you use the same device id in the config tool user field

**Comment by Jeffreyvanesch** [comment]
Unfortunately it also doesn't work for me, At startup it throws a unhandled exception at the connection box. If i continue with the application it doesn't show the connection box anymore.
Any chance that you have version 3.0? BTW, Do you have a similar problem ?
![image](https://cloud.githubusercontent.com/assets/25173470/26524839/a46259a4-4342-11e7-9f54-2bcda06752f9.png)

**Comment by justinmoninger** [comment]
I'm seeing the same issue as @Jeffreyvanesch .

**Comment by Whizkidzz** [comment]
@Jeffreyvanesch and @justinmoninger  the unhandled exception is raised because the program did not find any serial interface it could work with.
So you need a working serial interface to get rid of this error

**Comment by justinmoninger** [comment]
@Whizkidzz thanks.  Once I plugged in my Solaredge Gateway that error went away and it saw the USB as a serial device.  That said, the v2 software seems to be for the inverter only.  Do you have v3 available?  I'm wondering if that version can talk to the gateway.

**Comment by loboosh** [comment]
How to get this to work with no db9 connector on the inverter, have tried the rs232 terminals and the 485. 
I just want to have it on my office to monitor what is going on, with all the craziness in the world today do not want SE to control my stuff.
